### PR TITLE
Locales Catchup

### PR DESCRIPTION
**Pull request by TurtIeSocks**
Machine generated locales

@Fabio1988 @ReuschelCGN @Petap0w @bbdoc @pjorritsma 

@lenisko left out PL as requested, but there are some to catch up with :)

### CODE REVIEW

File: public/base-locales/de.json [hunk 1]
```diff
@@ -559,5 +559,9 @@
   "seen_nearby_cell": "in dieser S2 Zelle gesehen",
   "seen_lure_encounter": "getroffen am Lockmodul",
   "seen_lure_wild": "Spawn am Lockmodul",
-  "seen_wild": "Spawn in der Wildnis"
+  "seen_wild": "Spawn in der Wildnis",
+  "historic_rarity": "Historische Rarität",
+  "poi": "Sehenswürdigkeiten",
+  "300m_range": "300m Reichweite",
+  "lure_range": "Köderbereich"
```
Comment (Fabio1988) [inline public/base-locales/de.json:566]: Lockmodul Radius

File: public/base-locales/de.json [hunk 2]
```diff
@@ -559,5 +559,9 @@
   "seen_nearby_cell": "in dieser S2 Zelle gesehen",
   "seen_lure_encounter": "getroffen am Lockmodul",
   "seen_lure_wild": "Spawn am Lockmodul",
-  "seen_wild": "Spawn in der Wildnis"
+  "seen_wild": "Spawn in der Wildnis",
+  "historic_rarity": "Historische Rarität",
+  "poi": "Sehenswürdigkeiten",
```
Comment (Fabio1988) [inline public/base-locales/de.json:564]: It's related to Wayfarer... It's not even point of interest... It's called "Wayspots"

Comment (Fabio1988) [reply public/base-locales/de.json:564]: And also in english

https://niantic.helpshift.com/hc/en/21-wayfarer/section/167-how-to-review-wayspots/

**Comment by lenisko** [comment]
Leaving it here :) some of them been very hard to translate...

```json
{
  "ultra_beast": "Ultra Beast",
  "weather_options": "Opcje Pogody",
  "clickable_icon": "Ikona jest klikalna",
  "two_stops_till_next": "2 stopy do następnego",
  "one_stop_till_next": "1 stop do następnego",
  "no_more_gyms": "Koniec gymów",
  "light_map_border": "Jasna ramka mapy",
  "dark_map_border": "Ciemna ramka mapy",
  "cell_blocked": "Komórka zablokowana",
  "poi_color": "Kolor POI",
  "quest_condition": "Warunek zadania",
  "always_show_labels": "Zawsze pokazuj etykiety",
  "scan_areas_options": "Opcje obszarów skanowania",
  "historic_rarity": "Rzadkość historyczna",
  "poi": "Interesujące punkty",
  "300m_range": "Zasięg 300m",
  "lure_range": "Zasięg Lura",
  "tap_to_toggle": "Naciśnij, aby przełączyć",
  "power_up": "Power Up",
  "never_alt": "Nigdy",
  "seen_nearby_stop": "Widziany w pobliżu",
  "seen_nearby_cell": "Widziany w tej komórce",
  "seen_lure_encounter": "Spotkany z Lura",
  "seen_lure_wild": "Spawn z Lura",
  "seen_wild": "Dziki Spawn"
}
```

**Comment by bbdoc** [comment]
Should be ok... at least better than the automatic translations, even tough some are hard to translate 
```
  "scan_areas_options": "Options des zones de scan",
  "historic_rarity": "Rareté historique",
  "poi": "POI",
  "300m_range": "Portée 300m",
  "lure_range": "Distance des Leurres",
  "tap_to_toggle": "Appuyez pour basculer",
  "power_up": "Recharger",
  "never_alt": "Jamais",
  "seen_nearby_stop": "Vus à proximité",
  "seen_nearby_cell": "Vus dans cette cellule",
  "seen_lure_encounter": "Pokemon de Leurre",
  "seen_lure_wild": "Pokemon Sauvage de Leurre",
  "seen_wild": "Pokemon Sauvage"
```

**Pull request merged**